src/res/locales/app.he-IL.po

```diff
@@ -166,6 +166,11 @@ msgstr "הודעות בגלילה לאחורה"
 msgid "settings_formatting"
 msgstr "עיצוב נוסף"
 
+#: Nickname Colouring
+msg "settings_nick_colouring"
+msgstr: "צבע את כינויי המשתמשים בהודעות"
+
+
```

Comment:
Only add the translations to `app.dev.po`. The rest are generated by translations.kiwiirc.com by using `app.dev.po` as a template